src/algebra/algebra/tower.lean

```diff
@@ -123,6 +123,14 @@ ring_hom.ext $ λ _, rfl
 @[simp] lemma coe_to_alg_hom' : (to_alg_hom R S A : S → A) = algebra_map S A :=
 rfl
 
+@[simp] lemma _root_.alg_hom.commutes_of_tower (f : A →ₐ[S] B) (r : R) :
+  f (algebra_map R A r) = algebra_map R B r :=
+by rw [algebra_map_apply R S A r, f.commutes, ← algebra_map_apply R S B]
+
+@[simp] lemma _root_.alg_hom.comp_algebra_map_of_tower (f : A →ₐ[S] B) :
+  (f : A →+* B).comp (algebra_map R A) = algebra_map R B :=
+by ext; simp
```

Comment:
I'd be tempted to use the direct term-mode proof, which is presumably something like
```suggestion
@[simp] lemma _root_.alg_hom.comp_algebra_map_of_tower (f : A →ₐ[S] B) :
  (f : A →+* B).comp (algebra_map R A) = algebra_map R B :=
ring_hom.ext $ f.commutes_of_tower
```